src/TABLE.md

```diff
@@ -4,7 +4,7 @@
 | <span style="color:red;">Critical</span>       | Use appropriate origin checks                                         | Open access on extrinsics without checks may allow unauthorized actions that can compromise security                                           | Add access control checks to limit access to specific users or roles                                                     |
 | <span style="color:red;">Critical</span>       | Avoid unbounded iteration                                             | Unbounded iterations over large data structures can lead to resource exhaustion and potential denial of service                                | Implement limits or use a bounded storage map for these iterations                                                       |
 | <span style="color:red;">Critical</span>       | Unchecked input data                                                  | Lack of input validation can lead to unexpected behaviors and potential vulnerabilities                                                        | Validate input data before processing to ensure safe and predictable behavior                                            |
-| <span style="color:red;">Critical</span>       | Avoid unwrap usage inside runtime                                     | Using `unwrap()` or expect without proper error handling can lead to runtime panics and crashes                                                | Handle errors gracefully with `Result` or `Option` types to prevent panics                                               |
+| <span style="color:red;">Critical</span>       | Avoid unwrap usage inside runtime                                     | Using `unwrap()` or `expect` without proper error handling can lead to runtime panics and crashes                                                | Handle errors gracefully with `Result` or `Option` types to prevent panics                                               |
```

Comment:
```suggestion
| <span style="color:red;">Critical</span>       | Avoid unwrap usage inside runtime                                     | Using `unwrap()` or `expect()` without proper error handling can lead to runtime panics and crashes                                                | Handle errors gracefully with `Result` or `Option` types to prevent panics                                               |
```